frontend/src/app/game/game/game.component.scss

```diff
@@ -25,3 +25,11 @@
 .content {
   width: 100%;
 }
+
+.mobile-mitigation-panel {
+  position: fixed;
+  width: 100vw;
+  max-height: fit-content;
+  bottom: -0.5%;
+  background-color: #222222;
```

Comment:
Pls, do not use magic constants, we have e.g. `frontend/src/material-definitions.scss`.